src/common/commonutils/FileUtils.c

```diff
@@ -67,31 +67,79 @@ bool SavePayloadToFile(const char* fileName, const char* payload, const int payl
                     if (payload[i] != fputc(payload[i], file))
                     {
                         result = false;
-                        OsConfigLogError(log, "SavePayloadToFile: failed saving '%c' to '%s' (%d)", payload[i], fileName, errno);
+                        OsConfigLogError(log, "SaveToFile: failed saving '%c' to '%s' (%d)", payload[i], fileName, errno);
                     }
                 }
 
                 UnlockFile(file, log);
             }
             else
             {
-                OsConfigLogError(log, "SavePayloadToFile: cannot lock '%s' for exclusive access while writing (%d)", fileName, errno);
+                OsConfigLogError(log, "SaveToFile: cannot lock '%s' for exclusive access while writing (%d)", fileName, errno);
             }
-            
+
             fclose(file);

```

Comment:
Is it worth to check error code?

Reply:
Arguably, not. Unlikely to fail and if it fails, there is nothing else to do. Worst case scenario who gets this file next will find it empty/not flushed correctly.